### DISCUSSION TITLE
time remaining to battery is always 0

### DISCUSSION THREAD

**Discussion by J7044** [linuxdeepin/developer-center#3512] [Q&A | 问答板块] [answered]
### SUMMARY | 问题概要

whether the computer is loaded to maximum or minimum, I don't see a change in battery life on the screen

### STEPS TO REPRODUCE | 复现步骤

it is always visible even when the computer is just turned on

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

deepin 20.7V

**Comment by BLumia** [comment]
I cannot reproduce this issue on my machine, or maybe I didn't understand your meaning correctly :p

Did you mean the battery is always shows as zero (0) even thought it's fully charged?

**Comment by J7044** [comment]
yes, i have always 0 min remaining on my laptop
![Screenshot_select-area_20221129103010](https://user-images.githubusercontent.com/100805124/204491745-d4cf6663-1c9d-406f-8523-bba452b17886.png)

**Comment by J7044** [comment, marked as answer]
I figured out how to fix the problem, in the bios settings in the security section there was a check to enable display remaining battery time and it was off, it was enough to change it to make it work.

sorry for the wasting time.

**Reply by BLumia** [reply]
Glad you found out the reason. I've covert this to a discussion thread and marked this one as the answer, hope this could be helpful to other people who are facing the same issue as you :)
